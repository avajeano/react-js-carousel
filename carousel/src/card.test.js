import React from 'react';
import { render } from'@testing-library/react';
import Card from './Card';
import TEST_IMAGES from './_testCommon';


// smoke test
it("renders without crashing", function() {
    render(<Card />)
})

// snapshot test
it("matches snapshot", function () {
    const { asFragment } = render(<Card 
        caption="src/_testCommon.js"
        src="test1.com"
        currNum={1}
        totalNum={3}
        />)
    expect(asFragment()).toMatchSnapshot()
})