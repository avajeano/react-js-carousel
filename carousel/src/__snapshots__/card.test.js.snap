// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      src/_testCommon.js
    </h4>
    <img
      alt="src/_testCommon.js"
      class="Card-image"
      src="test1.com"
    />
    <small
      class="Card-small"
    >
      Image 1 of 3.
    </small>
  </div>
</DocumentFragment>
`;
